src/components/ClusterNodes/NodeDetails.tsx

```diff
@@ -0,0 +1,765 @@
+import React, { useState, useEffect } from 'react'
+import './clusterNodes.scss'
```

Comment:
I usually recommend including component-level CSS file at the end of all imports.

Reply:
Done